ddio_common/CMakeLists.txt

```diff
@@ -1,5 +1,34 @@
-set(HEADERS)
 set(CPPS
   ddio.cpp
-  key.cpp)
-add_library(ddio_common STATIC ${HEADERS} ${CPPS})
+  key.cpp
+  $<$<PLATFORM_ID:Darwin,Linux>:
+    lnxfile.cpp
+    lnxforcefeedback.cpp
+    lnxio.cpp
+    sdljoy.cpp
+    lnxkey.cpp
+    lnxkey_null.cpp
+    lnxmouse.cpp
+    lnxtimer.cpp
+    lnxcdrom.cpp
+    lnxkey_sdl.cpp
+  >
+  $<$<PLATFORM_ID:Windows>:
+    winfile.cpp
+    winforcefeedback.cpp
+    winio.cpp
+    winjoy.cpp
+    winkey.cpp
+    winmouse.cpp
+    wintimer.cpp
+  >
+)
+add_library(ddio_common STATIC ${CPPS})
```

Comment:
could we rename the lib simply `ddio` in the next edit?

Reply:
renamed to ddio.